script.js

```diff
@@ -0,0 +1,23 @@
+var url = 'http://api.icndb.com/jokes/random';
+	
+var button = document.getElementById('get-joke');
+
+//$(document).ready(function() { //Solution for willing
+	//getJoke();
+//});
+
+button.addEventListener('click', function() {
+	getJoke();
+});
+
+var paragraph = document.getElementById('joke');
+
+function getJoke() {
+	var xhr = new XMLHttpRequest();
+	xhr.open('GET', url);
+	xhr.addEventListener('load', function() {
+		var response = JSON.parse(xhr.response);
```

Comment:
after parse resposne and before you assign it to innerHTML check that value and joke properties exist in your response object